pom.xml

```diff
@@ -1485,5 +1485,45 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>java11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <java.version>11</java.version>
+                <hadoop.version>3.3.2</hadoop.version>
```

Comment:
Could be in properties...

Reply:
They are in properties? They can't go into the main properties section because properties with these names already exist there, these are in order to override those properties with different versions. Do you mean that we should have say a property called `java11.hadoop.version` in the main properties and then set `hadoop.version` to that in these properties? Like this:
```xml
<hadoop.version>${java11.hadoop.version}</hadoop.version>
```

Reply:
Yeah, would be better in the overall properties